var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -30,16 +30,21 @@ class Neovim(CMakePackage):
     version('0.2.1', sha256='9e2c068a8994c9023a5f84cde9eb7188d3c85996a7e42e611e3cd0996e345dd3')
     version('0.2.0', sha256='72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5')
 
+    variant('use_lua', default=False, description='use lua rather than luajit as lua language provider')
+
+    # depend on virtual, lua-luajit-openresty preferred
+    depends_on('lua-lang')
+    depends_on('luajit', when='~use_lua')
+
     depends_on('cmake@3.0:', type='build')
     depends_on('pkgconfig', type='build')
     depends_on('gettext', type=('build', 'link'))
-    depends_on('lua@5.1.0:5.1.9', type=('build', 'link'))
     depends_on('lua-lpeg', type='link')
     depends_on('lua-mpack', type='link')
-    depends_on('lua-bitlib', type='link')
+    depends_on('lua-bitlib', type='link', when='^lua')
```

Comment:
`lua-bitlib` is a requirement? Asking because I was not able to find it in the nevoim CMake.

Reply:
It should only be a requirement when using lua instead of luajit.  Also, it was already in there, I'm just restricting it to only the case when lua is selected instead of luajit.